cmd/validate/image.go

```diff
@@ -402,12 +403,10 @@ func validateImageCmd(validate imageValidationFunc) *cobra.Command {
 		"[DEPRECATED] write output to a file. Use empty string for stdout, default behavior")
 
 	cmd.Flags().BoolVarP(&data.strict, "strict", "s", data.strict,
-		"return non-zero status on non-successful validation")
+		"Return non-zero status on non-successful validation. Defaults to true. Use --strict=false to return a zero status code.")
 
 	cmd.Flags().StringVar(&data.effectiveTime, "effective-time", policy.Now, hd.Doc(`
-		Run policy checks with the provided time. Useful for testing rules with
-		effective dates in the future. The value can be "now" (default) - for
-		current time, "attestation" - for time from the youngest attestation, or
+		Run policy checks with the preturn non-zstation" - for time from the youngest attestation, or
```

Comment:
Should this be in this change? Seems off

Reply:
@zregvart don't know what happened shouldn't have been changed fixed it thank you.